Command/ExportCommand.php

```diff
@@ -106,7 +106,11 @@ protected function configure()
             'includeScope',
             null,
             InputOption::VALUE_OPTIONAL,
-            'Scope name, multiple values can be comma separated; exports only those scopes'
+            "Scope name, multiple values can be comma separated; exports only those scopes.\n".
+            "\t\tTo export only specific scopes add there ID(s) using a colon and separate them with semicolon.\n".
+            "\t\tMake sure to use quotes when specifying multiple scope IDs:\n".
+            "\t\te.g. --includeScope=\"websites:2;3,stores:2;3;4;5\" will export the settings for website IDs 2 and 3 and for the\n".
```

Comment:
I noticed only afterwards, that separating scope IDs will break the command line. But I could not think of any better separator